templates/snmpd.service-dropin-Debian.epp

```diff
@@ -1,6 +1,18 @@
 ###
 ### File managed by Puppet
 ###
+[Unit]
```

Comment:
This doesn't look like a drop in anymore. Why do you need to define all these things? If it really needs a different file name.

Reply:
Swapping to unit file